skydriver/k8s.py

```diff
@@ -336,6 +338,7 @@ def _get_token_from_keycloak(
     def get_env(
         rest_address: str,
         scan_id: str,
+        max_reco_time: int | None,
```

Comment:
The method is static because it's only used by `__init__()`. Making this an instance method would require a certain amount of implicit state before calling it--and that's not my favorite pattern. I could rename the method `_get_env()`.

Reply:
As long as you have a good reason for it.  It just seems a bit odd since we're now getting multiple things in that don't really come from the env.

Reply:
it's more like "get the k8s jobs' environment" rather than read the current